blocks/test/fixtures/core__chat.html

```diff
@@ -0,0 +1,13 @@
+<!-- wp:core/chat {"compact":false} -->
+Pascal: Alexa, open the window
+Alexa: I'm not quite sure how to help you with
+Pascal: Siri, tell me a joke
+Siri: Pascal, get Siri-ous. Ha ha!
+Pascal: OK, Google. Now it's your turn!
+Google Home: Alexa, tell Siri to tell Pascal a better joke.
+Pascal: Ugh!
+Okay, that's enough. No internet for you anymore!
+Alexa: 💔
+Google Home: 😔
+Siri: 📴
```

Comment:
I think there's a problem here unless I'm missing something.

If I understand the code, this value will be parsed and reserialized differently (using HTML ...).
What happens if I try to re-edit the block again? The parser will fail?
Anyway, The source value and the serialized value should be the same (aside some spacing differences maybe) which makes me think we should store the raw value in the comment.